src/main/java/com/massivecraft/factions/P.java

```diff
@@ -28,7 +33,14 @@
 
 
 public class P extends MPlugin {
-
+	
```

Comment:
There's a tab on this line, which should be removed. And all the code in this block is using tabs. Fix that back to spaces.